### PR TITLE
fix(histogram): display correct percentile value instead of formula

### LINKED ISSUE #17550: [histogram] percentile value in tooltip shows formula instead of value
#### How to reproduce the bug

1. Go to 'FCC New Coder Survey 2018' sample dashboard.
2. Check the tooltip in histogram chart 'Age distribution of respondents'

### Expected results

In tooltip, percentile value should be shown.

### Actual results

In tooltip, percentile value formula is shown.

#### Screenshots

image

### Environment

(please complete the following information):

- browser type and version: Chrome 95.0.4638.69
- superset version: `master`
- python version: `3.7`
- node.js version: `14.15.5`
- any feature flags active: default

### Checklist

Make sure to follow these steps before submitting your issue - thank you!

- [x] I have checked the superset logs for python stacktraces and included it here as text if there are any.
- [x] I have reproduced the issue with at least the latest released version of superset.
- [x] I have checked the issue tracker for the same issue and I haven't found one similar.

### Additional context

cc: @villebro

### PR DESCRIPTION
**Pull request by hoangminhquan-lhsdt**
### SUMMARY

This PR addresses issue #17550 regarding histogram percentile displaying the formula instead of the value. The long formula also causes the entire tooltip box to be long, which possibly leads to the entire tooltip being hidden to the side of the plot.

### BEFORE

Screenshot from 2022-01-19 09-26-36

### AFTER
Screenshot from 2022-01-19 09-34-39

### TESTING INSTRUCTIONS

- Open any histogram chart, "FCC 2018 Survey's Age distribution of respondents" is an included example.
- Hover the mouse pointer over any column to see the percentile value in the tooltip.

### ADDITIONAL INFORMATION

- [x] Has associated issue: Fixes #17550 
- [x] Required feature flags: Disables prettier linting in `superset-frontend/plugins/legacy-plugin-chart-histogram/src/Histogram.jsx` to prevent ERROR related to line length during `npm build`.
- [x] Changes UI: no
- [x] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351)): no
  - [x] Migration is atomic, supports rollback & is backwards-compatible: N/A
  - [x] Confirm DB migration upgrade and downgrade tested: N/A
  - [x] Runtime estimates and downtime expectations provided: N/A
- [x] Introduces new feature or API: no
- [x] Removes existing feature or API: no

### CODE REVIEW

File: superset-frontend/.prettierignore [hunk 1]
```diff
@@ -26,3 +26,4 @@ CHANGELOG.md
 *-topo.json
 temporary_superset_ui/
 storybook-static/
+plugins/legacy-plugins-chart-histogram/src/Histogram.jsx
```
Comment (nytai) [inline superset-frontend/.prettierignore:29]: why is this necessary. Can you just run prettier or fix the error instead of ignoring this file?

Comment (hoangminhquan-lhsdt) [reply superset-frontend/.prettierignore:29]: When I first ran Prettier on the file, it didn't change anything. I suspect it was because either I misconfigured Prettier on my local machine, or Prettier confused and couldn't reformat. I will try to configure it again.

**Review by nytai — APPROVED** [review]
